DESCRIPTION

```diff
@@ -48,11 +48,12 @@ Imports:
 Suggests:
     knitr,
     lubridate (>= 1.7.4),
-    pharmaversesdtm (>= 0.2.0),
+    pharmaversesdtm,
     rmarkdown,
     stringr (>= 1.4.0),
     testthat (>= 3.0.0),
     tibble
+Remotes: pharmaverse/pharmaversesdtm@admiralpeds_data_migration
```

Comment:
@zdz2101 this bit can be removed and you could get this and https://github.com/pharmaverse/pharmaversesdtm/pull/115 ready for review now, and think should be good timing for me and @Fanny-Gautier to take a look in the coming weeks

Reply:
I think the steps go:
1. merge the pharmaversesdtm PR first
2. remove this line
3. see if this PR still works with the removal of `pharmaversdtm::`, because I think it'll still rely on a remotes until the pharmaversesdtm package is re-released to a new version, as I'm still running into namespacing issues without it

Reply:
Plus Step 4) add an `adsl_peds` and `advs_peds` to pharmaverseadam. we can make an issue for this later after all these steps

Reply:
As for `adsl_peds` and `advs_peds` it seems there was some ongoing discussion about what is appropriate to house in there as the datasets for that package are made from the templates in `admiral` see [discussion from this issue](https://github.com/pharmaverse/admiral/issues/2490#issuecomment-2284097451)

Reply:
I think they mean any admiral pks templates incl extensions here, as they do have ADaMs created from ophtha, onco, vaccine templates for example at https://github.com/pharmaverse/pharmaverseadam/tree/main/data